### PR TITLE
Add a README and examples for invoking test harness by hand

### PR DESCRIPTION
**Pull request by cbowman0**
It's been a little while since I wrote some tests and had to remember how to invoke the test harness and verify the test coverage, so I wrote it down.  Wording and everything open for discussion/adjustment and I may not be describing the optimal ways to do any of this.

* Add README for how the Django-based tests are run.
* Also adjust the coverage invocation so that manual runs can pass in specific test classes or test methods to run during debugging/testing.

### CODE REVIEW

File: webapp/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Graphite-Web Test Framework
+
+## Overview
+
+These Graphite-Web unit tests are for the Django portions of the graphite-web code base.  They utilize the Django test framework and run via `manage.py test`.
+
+For pull-requests, the tests run inside of Travic-ci using Python tox (https://pypi.python.org/pypi/tox) to invoke the tests across all the environments defined in tox.ini.
```
Comment (deniszh) [inline webapp/tests/README.md]: Travis-ci ?

**Comment by deniszh** [comment]
Very cool, indeed!
Thanks, @cbowman0 !

File: webapp/tests/README.md (hunk 1 above)
Comment (cbowman0) [reply webapp/tests/README.md]: Yep.  Will fix.

Comment (cbowman0) [reply webapp/tests/README.md]: Thanks for reading through it.

**Comment by cbowman0** [comment]
Additionally to this quick documentation, I was thinking it would be good to describe/document what dependencies are needed to have an environment where the tests would run.  I believe they currently need Python tox, coverage, a redis server on localhost and a mysql server on localhost.   Maybe an Ubuntu/CentOS VM?  Are there other ways that are simpler/easier?

**Comment by DanCech** [comment]
Actually if you run tox with no additional parameters right now I think it's going to try and run a huge number of different environments (check out the output of `tox -l`) so we should probably be more prescriptive there.  I usually run `py27-django111-pyparsing2` and `py35-django111-pyparsing2` locally along with `lint`.

As far as requirements, those only need a local redis server.  If you want to run `-mysql` or `-postgresql` locally then you'll need to have servers running , database users, etc.  For 99.9% of development that isn't required though, only if you're hacking on the tag db.

**Comment by cbowman0** [comment]
OK.  I changed the words to highlight the environments to test in and added them to the examples.  I'm done changing the words for now.

File: webapp/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+A set of minimum environments to run tests against are:
+`py27-django111-pyparsing2`
+`py35-django111-pyparsing2`
+`lint`
+
+Passing multiple environments to tox can be done by separating with commas:
+`tox -e py27-django111-pyparsing2,py35-django111-pyparsing2,lint`
+
+To run only a specific test suite run (aka. run the file tests/test_finders.py):
+`tox -e py27-django111-pyparsing2,py35-django111-pyparsing2,lint-- tests.test_finders`
```
Comment (DanCech) [inline webapp/tests/README.md]: missing space?

Comment (cbowman0) [reply webapp/tests/README.md]: Yes.

**Pull request merged**